api/v1/swagger.yaml

```diff
@@ -1330,7 +1330,6 @@ paths:
                     "Collector", "Subject", etc.
               required:
                 - details
```

Comment:
What about making details not required and defaulting it to {}; similar to what we did for the "loggedAt" column?

Reply:
OK feel free to make that change here in swagger, adding default value

Reply:
OK, will do it.